playbooks/configure-stack.yml

```diff
@@ -6,7 +6,8 @@
     - name: copy over docker compose
       copy:
         src: docker-compose
-        dest: /home/appdev/deploy
+        dest: /home/appdev/

```

Comment:
may want to check if trailing slash is proper here

Reply:
ya i checked by deploying on a test server, we're gucci